pom.xml

```diff
@@ -112,12 +112,14 @@
 		<benchmarkjar.name>benchmarks</benchmarkjar.name>
 		<javac.target>1.8</javac.target>
 		<slf4j.version>1.7.21</slf4j.version>
-		<log4j2.version>2.6.2</log4j2.version>
+		<log4j2.version>2.1</log4j2.version>
```

Comment:
u may have to shade out the logging from Storm, so that Storm picks the
project log4j version.

On Fri, Aug 19, 2016 at 3:33 PM, Chris Harris <notifications@github.com>
wrote:

> In pom.xml
> <https://github.com/apache/incubator-pirk/pull/74#discussion_r75539610>:
>
> > @@ -112,12 +112,14 @@
> >  		<benchmarkjar.name>benchmarks</benchmarkjar.name>
> >  		<javac.target>1.8</javac.target>
> >  		<slf4j.version>1.7.21</slf4j.version>
> > -		<log4j2.version>2.6.2</log4j2.version>
> > +		<log4j2.version>2.1</log4j2.version>
>
> When I ran it on the cluster I would get
>
> Exception in thread "main" java.lang.IllegalAccessError: tried to access method org.apache.logging.log4j.core.lookup.MapLookup.newMap(I)Ljava/util/HashMap; from class org.apache.logging.log4j.core.lookup.MainMapLookup
>     at org.apache.logging.log4j.core.lookup.MainMapLookup.<clinit>(MainMapLookup.java:37)
>
> It seems there's a conflict w/ the logging from Storm and this was my
> quick/lazy way to fix it. I'll try to dig into it some more.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apache/incubator-pirk/pull/74/files/2d2abe1b90891737204bbded9d9d6f7b3fa2f1ff#r75539610>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AC_fop2oPlieyU9QVfFj7SZxRFl0obpFks5qhgUbgaJpZM4Jov-l>
> .
>